spanner_orm/field.py

```diff
@@ -222,18 +241,22 @@ def validate_type(value: Any) -> None:
 
   @staticmethod
   def matches(type: str) -> bool:
-    return type.startswith('ARRAY<STRING(') and type.endswith(')>')
+    val = re.findall('ARRAY<(.*)>', type)
```

Comment:
(nit) ditto about fullmatch

Reply:
Since we use regex, we'll not leverage String.matches but just do the matching here as needed.